csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -611,6 +612,13 @@ std::unique_ptr<ReductionParams> innerOuterPersistentHeuristic(
     rparams->block_dim_iter_dom = ParallelType::TIDy;
   } else {
     rparams->block_dim_inner_reduction_extra = ParallelType::TIDy;
+    rparams->static_bdimx = true;
+    rparams->static_bdimy = true;
```

Comment:
I think here we need to have all block dimensions be static, since `bdimz == 1`?

Reply:
Ha, I see it now. Thanks for bearing with me here.